Adding Multi channel modems

AnneThKu opened the pull request:
Below changes are done:
- Added `Acoustic Data Acquisition system` in `Products` page.
- Updated `Water Quality Monitoring` to `Subnero Water Assessment Network`.
- Added `aRecorder` and `multi-channel modem` to DAQ category.
- Added `multi-channel modem` to Silver edition section as well.
- Added a line for availability of customised products in the modems page.

AnneThKu committed:
feat(modem):Adding multi-channel modems

AnneThKu committed:
feat(modem):Adding multi-channel modems

AnneThKu committed:
feat(modem):Adding multi-channel modems

AnneThKu committed:
feat(modem):Adding multi-channel edition

AnneThKu committed:
feat(modem):Adding multi-channel edition

AnneThKu committed:
feat(modem):Adding multi-channel edition

AnneThKu committed:
feat(modem):Adding multi-channel edition

AnneThKu committed:
feat(products):Updating meta description for SEO

AnneThKu committed:
feat(products):Updating meta description for SEO

AnneThKu committed:
feat(products):Updating meta description for SEO

AnneThKu committed:
fix(solutions):Updated meta description for SEO

AnneThKu committed:
fix(solutions):Updated meta description for SEO

AnneThKu committed:
fix(solutions):Updated meta description for SEO

AnneThKu committed:
feat(daq):Adding category for daq in products

AnneThKu committed:
feat(daq):Adding category for daq in products

AnneThKu committed:
feat(daq):Adding category for daq in products

AnneThKu committed:
feat(daq):Adding category for daq in products

AnneThKu committed:
fix(daq):Removing commented lines

AnneThKu committed:
fix(daq):Removing commented lines

AnneThKu committed:
fix(daq):Removing commented lines

AnneThKu committed:
fix(daq):Removing commented lines

AnneThKu commented:
Noting down the broken links.
- 2018-05-27-UnetStack-Newsletter.md
  - http://www.oceans18mtsieeekobe.org/social-events/
  - http://www.oceans18mtsieeekobe.org/tutorial-information/
- 2019-06-11-UnetStack-Newsletter.md
  - http://www.oceans19mtsieeemarseille.org/ 
- 2019-10-15-Subnero-expands-in-china.md
  - http://www.smartseatech.com/en/index.aspx 
- 2019-10-29-Subnero-at-iOcean-Sanya.md
  - http://www.cso.org.cn/ggxx/2019/0926/2816.html 
  - https://singapore20.oceansconference.org/

notthetup commented:
@AnneThKu I've raised an issue for broken links https://github.com/subnero1/subnero1.github.io/issues/119 

Maybe you can add the ones you found to the same one.

Florie25 committed:
Create 2020-10-12-Global-Oceans-2020.md

Florie25 committed:
Update 2020-10-12-Global-Oceans-2020.md

manuignatius committed:
Create 2020-12-16-UnetStack-paper-on-Residual-Energy-Model.md

Florie25 committed:
Update 2020-12-16-UnetStack-paper-on-Residual-Energy-Model.md

Florie25 committed:
Create 2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

AnneThKu committed:
feat(swan):Updated white paper

AnneThKu committed:
Merge pull request #136 from subnero1/white-paper

feat(swan):Updated white paper

manuignatius committed:
Update index.md

manuignatius committed:
Merge pull request #137 from subnero1/Production-Engineer-JD

Update careers page to include production engineer JD

Florie25 committed:
Update 2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Florie25 committed:
Update 2020-10-12-Global-Oceans-2020.md

Florie25 committed:
Update _posts/2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Add files via upload

Florie25 committed:
Update 2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Florie25 committed:
Update _posts/2020-10-12-Global-Oceans-2020.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-10-12-Global-Oceans-2020.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-10-12-Global-Oceans-2020.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Add files via upload

Florie25 committed:
Update 2020-10-12-Global-Oceans-2020.md

Florie25 committed:
Update 2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

Florie25 committed:
Add files via upload

Florie25 committed:
Update 2020-10-12-Global-Oceans-2020.md

AnneThKu committed:
fix(pulse):Renamed Oceans 2020 thumbnail

Florie25 committed:
Update 2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

AnneThKu committed:
fix(pulse):Changed thumbnail image for oceans 2020 post

Florie25 committed:
Create 2020-11-19-Subnero-at-SIWW-online.md

Florie25 committed:
Add files via upload

AnneThKu committed:
fix(pulse): Changed the image size for oceans 2020 post

Florie25 committed:
Update 2020-11-19-Subnero-at-SIWW-online.md

Florie25 committed:
Create 2021-01-02-Build-the-future-podcast.md

Florie25 committed:
Add files via upload

Florie25 committed:
Merge pull request #138 from subnero1/Nitk-presentations

NITK presentations

Florie25 committed:
Merge pull request #139 from subnero1/Oceans-Global

Oceans global

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 146.10kb -> 135.64kb (7.16%)

/images/pulse-thumbnail-NITK-2020.jpg -- 97.62kb -> 89.96kb (7.84%)
/images/pulse-Oceans-2020.jpg -- 40.09kb -> 37.31kb (6.93%)
/images/pulse-thumbnail-oceans2020.jpg -- 8.39kb -> 8.36kb (0.34%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

notthetup committed:
Merge pull request #142 from subnero1/imgbot

[ImgBot] Optimize images

dependabot[bot] committed:
Bump nokogiri from 1.10.10 to 1.11.1

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.10.10 to 1.11.1.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.10.10...v1.11.1)

Signed-off-by: dependabot[bot] <support@github.com>

notthetup committed:
Merge pull request #143 from subnero1/dependabot/bundler/nokogiri-1.11.1

Bump nokogiri from 1.10.10 to 1.11.1

Florie25 committed:
Update _posts/2020-11-19-Subnero-at-SIWW-online.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-11-19-Subnero-at-SIWW-online.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2020-11-19-Subnero-at-SIWW-online.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Add files via upload

Florie25 committed:
Update 2020-11-19-Subnero-at-SIWW-online.md

Florie25 committed:
Update 2020-11-19-Subnero-at-SIWW-online.md

Florie25 committed:
Update 2020-11-19-Subnero-at-SIWW-online.md

Florie25 committed:
Update _posts/2021-01-02-Build-the-future-podcast.md

Co-authored-by: AnneThKu <42397354+AnneThKu@users.noreply.github.com>

Florie25 committed:
Update _posts/2021-01-02-Build-the-future-podcast.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

Florie25 committed:
Update 2021-01-02-Build-the-future-podcast.md

AnneThKu committed:
fix(pulse): Fixed image size and added target blank for SIWW link

Florie25 committed:
Merge pull request #141 from subnero1/podcast

Podcast

ImgBotApp committed:
[ImgBot] Optimize images

/images/pulse-thumbnail-build-the-future.png -- 328.74kb -> 259.21kb (21.15%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

notthetup committed:
Merge pull request #144 from subnero1/imgbot

[ImgBot] Optimize images

Florie25 committed:
Merge pull request #140 from subnero1/SIWW_online

Siww online

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 891.74kb -> 635.75kb (28.71%)

/images/pulse-thumbnail-siww-online.png -- 784.65kb -> 535.94kb (31.7%)
/images/pulse-siww-online-2020.jpg -- 107.10kb -> 99.81kb (6.81%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

notthetup committed:
Merge pull request #145 from subnero1/imgbot

[ImgBot] Optimize images

manuignatius committed:
refactor: Updated content of product, modem and monitoring pages

manuignatius committed:
refactor: Updated content of product, modem and monitoring pages

manuignatius committed:
refactor: Updated content of product, modem and monitoring pages

manuignatius committed:
WIP: Updating multichannel modem pages

manuignatius committed:
WIP: Updating multichannel modem pages

manuignatius committed:
WIP: Updating multichannel modem pages

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

AnneThKu committed:
Merge branch 'multi-channel' of https://github.com/subnero1/subnero1.github.io into multi-channel

AnneThKu committed:
feat(prodcuts):Adding battery pack info page

AnneThKu committed:
feat(prodcuts):Adding battery pack info page

notthetup commented on products/accessories.md:
```diff
@@ -0,0 +1,64 @@
+	<h1 class='thin' style='text-align: center'>Optional Accessories and Upgrades</h1>
+	<div class='eyXpDN'>
+		<div class='cmXrEt'>
+			<!-- <h1 class='thin' style='text-align: center'>Platinum Edition</h1> -->
```
```suggestion
```

notthetup commented on products/accessories.md:
```diff
@@ -0,0 +1,64 @@
+  <div class='four spacing'></div>
+</div>
+
+<!--<div class='full' style='background:#F5F5F5'>-->
```
```suggestion
```

manuignatius committed:
Update products/accessories.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius committed:
Update products/accessories.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius committed:
Update products/accessories.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius committed:
Update products/accessories.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius commented:
This branch is still WIP, the PR was raised a bit too early. Will re-request for a review once the updates are done.

AnneThKu committed:
Merge branch 'multi-channel' of https://github.com/subnero1/subnero1.github.io into multi-channel

AnneThKu committed:
feat(products): Add batterypack.md

AnneThKu committed:
feat(products): Add batterypack.md

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

manuignatius committed:
WIP

AnneThKu committed:
feat(products):Add + for collapse

AnneThKu committed:
feat(products):Add + for collapse

notthetup commented:
Converted to draft. Convert to "ready" when you're ready for review @manuignatius

AnneThKu committed:
feat(products): Adding collapisble feature

AnneThKu committed:
feat(products): Adding collapisble feature

manuignatius committed:
WIP

manuignatius committed:
WIP

AnneThKu committed:
feat(products): Multiple images for accessories

AnneThKu committed:
feat(products): Multiple images for accessories

AnneThKu committed:
feat(includes):Adding display.html

AnneThKu committed:
feat(includes):Adding display.html

manuignatius committed:
Updated to check alignment

manuignatius committed:
Updated to check alignment

manuignatius committed:
WIP: Updated content on all modem pages

manuignatius committed:
WIP: Updated content on all modem pages

manuignatius committed:
WIP: Updated all pictures for existing modems

manuignatius committed:
WIP: Updated all pictures for existing modems

manuignatius committed:
WIP: Updated technical specifications to match with latest tech specs

manuignatius committed:
WIP: Updated technical specifications to match with latest tech specs

manuignatius committed:
WIP: Updated battery pack page

manuignatius committed:
WIP: Updated battery pack page

manuignatius committed:
WIP: Added separate pages for ec, sc and mc modems for silver ed. Platinum ed pending

manuignatius committed:
WIP: Added separate pages for ec, sc and mc modems for silver ed. Platinum ed pending

manuignatius committed:
WIP: Updated layout of silver and research edition modems

manuignatius committed:
WIP: Updated layout of silver and research edition modems

manuignatius committed:
WIP: updated options page

manuignatius committed:
WIP: updated options page

manuignatius committed:
WIP: updated options page

manuignatius committed:
WIP: updated options page

manuignatius committed:
Update 2020-12-16-UnetStack-paper-on-Residual-Energy-Model.md

manuignatius committed:
WIP: Updated platinum modem pages

manuignatius committed:
WIP: Updated platinum modem pages

manuignatius committed:
WIP: Optional accessories page updates

manuignatius committed:
WIP: Optional accessories page updates

manuignatius committed:
coprocessor, battery packs done

manuignatius committed:
coprocessor, battery packs done

manuignatius committed:
WIP: Added thumbnail for accessories

manuignatius committed:
WIP: Added thumbnail for accessories

manuignatius committed:
WIP: Added sensors and storage pages

manuignatius committed:
WIP: Added sensors and storage pages

manuignatius committed:
WIP: Updated pictures and contact for quote position

manuignatius committed:
WIP: Updated pictures and contact for quote position

manuignatius committed:
Updated coprocessor images and interfaces page

manuignatius committed:
Updated coprocessor images and interfaces page

manuignatius committed:
WIP: Added cables, clamps and hulls pages

manuignatius committed:
WIP: Added cables, clamps and hulls pages

AnneThKu committed:
fix(products): Extended accessories image section

AnneThKu committed:
fix(products): Extended accessories image section

AnneThKu committed:
fix(products): Changing width for accessories image section

AnneThKu committed:
fix(products): Changing width for accessories image section

AnneThKu committed:
fix(products): Fixing image info list

AnneThKu committed:
fix(products): Fixing image info list

AnneThKu committed:
fix(products): Fixing product images

AnneThKu committed:
fix(products): Fixing product images

AnneThKu committed:
fix(products): Fixed image section for accessory pages

AnneThKu committed:
fix(products): Fixed image section for accessory pages

AnneThKu committed:
fix(products):Reverted collapsible change for Tech specs

AnneThKu committed:
fix(products):Reverted collapsible change for Tech specs

manuignatius committed:
Updated all accessories pages, pending pictures of right angled bulkhead connectors

manuignatius committed:
Updated all accessories pages, pending pictures of right angled bulkhead connectors

manuignatius committed:
Updated banner images for battery page

manuignatius committed:
Updated banner images for battery page

AnneThKu committed:
Merge branch 'multi-channel' of https://github.com/subnero1/subnero1.github.io into multi-channel

manuignatius committed:
Updated cables, right angled bulkhead connectors and spell check. Pending items: connect and uw cable model numbers

manuignatius committed:
Updated cables, right angled bulkhead connectors and spell check. Pending items: connect and uw cable model numbers

manuignatius committed:
feat: Ready for review

manuignatius committed:
feat: Ready for review

manuignatius commented:
Review required
- All pages under "Wireless Networked Communications" page.
- Changes
   - Updated platinum edition and silver edition modems pages
   - Updated SC from surface to standalone
   - Removed node configuration
   - Added multi-channel modems
   - Added optional upgrades and accessories section
   - Updated modem brochure and technical specs

AnneThKu commented:
Travis failing as the below mentioned links are broken:
Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff
IoT Weekly Podcast (19-Jan-2018) - https://gearforgesoftware.com/iot-weekly-podcast/iot-weekly-episode-8-podcast/
Robot SWANs to monitor water quality in Singapore (15-Jan-2018) - https://www.channelnewsasia.com/404

notthetup commented:
@AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?

notthetup commented:
Here is one.. ![web archive org_web_20201220173152_https___www channelnewsasia com_news_singapore_robot-swans-reservoirs-monitor-water-quality-pub-9861686_](https://user-images.githubusercontent.com/553140/111263003-1f987680-8660-11eb-9bac-2dff3f60f1c5.png)

notthetup commented:
1. This banner is very strange. The white space in the middle with the modems all the way to the edge looks like something is broken..

![Screen Shot 2021-03-16 at 21 14 03](https://user-images.githubusercontent.com/553140/111315240-272a4080-869d-11eb-9fba-e4add663e00c.jpg)

 2. Instead of just saying "D", we can say `D (IEC R20)` which is the official standard name for D-cells
![Screen Shot 2021-03-16 at 21 14 33](https://user-images.githubusercontent.com/553140/111315431-55a81b80-869d-11eb-88d9-5d6f39afe7fc.jpg)

3. These random stock photos of storage mediums look ugly. If the photos of SDCards are repetitive, we can just put one photo on the page.
![Screen Shot 2021-03-16 at 21 15 33](https://user-images.githubusercontent.com/553140/111315525-6d7f9f80-869d-11eb-8aaf-7b952ef86671.jpg)

4. Also the mouse over blurring+zooming effect on all the photos in that array makes them look worse.

5. Strange grey line below the photo of the platinum edition modem. Also, why is it horizontal?
![Screen Shot 2021-03-16 at 21 16 13](https://user-images.githubusercontent.com/553140/111315887-c2bbb100-869d-11eb-8b5e-0378e97ab13d.jpg)

6. The ziplock with handwritten stuff looks very unprofessional. We should print a label or not have any label.
![Screen Shot 2021-03-16 at 21 16 42](https://user-images.githubusercontent.com/553140/111316010-dbc46200-869d-11eb-9b65-5a01bb5647b0.jpg)

7. What is this photo supposed to be?
![Screen Shot 2021-03-16 at 21 25 55](https://user-images.githubusercontent.com/553140/111316590-5b523100-869e-11eb-9500-9cadce70b70f.jpg)

8. This array has repeated photos..
![Screen Shot 2021-03-16 at 21 26 01](https://user-images.githubusercontent.com/553140/111316800-88064880-869e-11eb-9879-665f8942b275.jpg)

9. Some sections have +/- signs on the left and some on the right. Some can be folded, some can't. We should be consistent.
![Screen Shot 2021-03-16 at 21 30 09](https://user-images.githubusercontent.com/553140/111317128-d1569800-869e-11eb-8838-20d296a3f512.jpg) 
![Screen Shot 2021-03-16 at 21 14 33](https://user-images.githubusercontent.com/553140/111317147-d582b580-869e-11eb-976f-f567fbb6b306.jpg)

10. Some pages have a pink? background, others are white. We should be consistent.

notthetup commented on _includes/collapse.html:
```diff
@@ -0,0 +1,20 @@
+<!-- Start Collapse.html -->
```
Can we use a pure-css based collapsible instead?

https://www.digitalocean.com/community/tutorials/css-collapsible

AnneThKu replied:
Changed to css-collapsible in https://github.com/subnero1/subnero1.github.io/pull/120/commits/aba146c0e2624dd6af0b635d60c3f3681ffe30bd and  https://github.com/subnero1/subnero1.github.io/pull/120/commits/bab7b4bc084ab549d7cccbfe82cb8f0c43f73534.

notthetup commented on _posts/2020-10-12-Global-Oceans-2020.md:
```diff
@@ -0,0 +1,35 @@
+---
```
Why are all the pulse posts a part of this pull request?

manuignatius committed:
fix: Updated D to full spec

manuignatius committed:
fix: Updated D to full spec

notthetup commented on products/accessories.md:
```diff
@@ -0,0 +1,177 @@
+	<div class='eyXpDN'>
+		<div class='cmXrEt'>
+			<h1 class='thin' style='text-align: center'>Battery Packs</h1>
+			<ul class="gfXsQG">
```
What does this class `gfXsQG` do?

manuignatius commented:
1. Will update the banner to reduce the white space
2. Updated D to D (IEC R20)
3. Will need @AnneThKu 's help with this. As of now, there is a limitation of having 4 images in all pages. Hence the random stock images. Would be great if we do not have such a limitation, not sure how hard is it in html/css/javascript. Seems pretty hard.
4. Needs @AnneThKu  to fix this
5. The grey line is the line that comes during mouse over? Image can be updated to vertical, but would like to stick with 3 images if we can have different number of images in a page
6. Will remove the zip lock from the image with only clamps in there
7. Right angled bulk head connector (similar image in the original subconn product page). We can consider changing it to a right angled connector with cable connected. Again, would be great if we can just have 2 images in this page.
8. In general, since there is a constraint of exact 4 images, some of the images are repeated. If we can fix to any number of images between 1 - 4, it will be easier
9. The + sign is for @AnneThKu  to add folding to the page. I think these are pending. If we are not able to do it (seems it is difficult), I will remove those '+' signs and keep all the text visible on page load (which ideally I would like to avoid)
10. The modem product pages are all "pink/grey" bg but the "accessories" pages are all white based on the template. @AnneThKu can you help update all accessories page to "pink/grey"?

notthetup commented on products/accessories.md:
```diff
@@ -0,0 +1,177 @@
+		<div class='cmXrEt'>
+			<h1 class='thin' style='text-align: center'>Battery Packs</h1>
+			<ul class="gfXsQG">
+				<li class="fuqHMA">
```
Same with class `fuqHMA`?

manuignatius replied:
Not sure I understand :(

notthetup replied:
It's best practice to give CSS classes names that make sense. For example `class="col-xs-6` which implies this is a `column` of `xs` size and 6 points in width. But `fuqHMA` doesn't mean anything.

AnneThKu replied:
These classes were taken from the reference provided during the revamping of the product listing page and were used as it is. I have raised an issue to rename all the classes (https://github.com/subnero1/subnero1.github.io/issues/148).

notthetup commented on products/batterypack2.md:
```diff
@@ -0,0 +1,113 @@
+<div class='full tall' style='background-image: url({{site.baseurl}}/{{page.banner}});'>
+  <div class='row'>
+    <div class='large-12 columns'>
+      <!-- {% include section-header.html title=page.title tagline=page.tagline color=page.title_color class="big" %} -->
```
```suggestion
```

notthetup commented:
> The grey line is the line that comes during mouseover? 

I see the grey line under some images all the time. Something about screen size?

notthetup commented:
I tried with 3 images, and I get this. Seems OK.
 
![Screen Shot 2021-03-16 at 21 52 04](https://user-images.githubusercontent.com/553140/111320469-f3054e80-86a1-11eb-8ef3-9e2c1a7e7a72.jpg)

manuignatius committed:
Update products/batterypack2.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius committed:
Update products/batterypack2.md

Co-authored-by: Chinmay Pendharkar <chinmay@subnero.com>

manuignatius commented:
> I tried with 3 images, and I get this. Seems OK.
> 
> ![Screen Shot 2021-03-16 at 21 52 04](https://user-images.githubusercontent.com/553140/111320469-f3054e80-86a1-11eb-8ef3-9e2c1a7e7a72.jpg)

The size of the images changes if the number of images are less. Try resizing the page or a different screen size, you will be able to see it. Basically, the thumbnails starts to become bigger and bigger.

manuignatius committed:
fix: Updated battery banner image

manuignatius committed:
fix: Updated battery banner image

manuignatius commented:
1. Updated battery image

Will need @AnneThKu help with
- Fix collapsing in the modem pages
- Fixing bg color in accessories page to match with modem page
- Having random number of images in accessories page without changing size

Once these are done, I can fix the rest of the comments.

manuignatius committed:
fix: updated clamp picture to remove the hand written ziploc

manuignatius committed:
fix: updated clamp picture to remove the hand written ziploc

mchitre reviewed (approved):
@manuignatius
Overall much better than what we have, so I recommend going ahead with this.

A critique, however, is that the content doesn't seem to flow smoothly, in general. While nothing incorrect with the text, the overall pages feel like disconnected text blurbs. At some point, we might need a more holistic look at the overall flow of the site.

For example, if you look at the multichannel modem page, it starts with "Subnero underwater modems provide options for customization and extension at many levels." But this is a standard configuration, so there's no customization involved! Then after key features, you have these 3 inset boxes which feel like they're meant to clickable to get more info, but they're not. And they just repeat what the last 3 bullets of features say.

manuignatius committed:
Merge pull request #134 from subnero1/unetstack-paper

Create 2020-12-16-UnetStack-paper-on-Residual-Energy-Model.md

AnneThKu committed:
fix(accessories): Fix for background color and collapsible in batterypack files

AnneThKu committed:
fix(accessories): Fix for background color and collapsible in batterypack files

AnneThKu committed:
fix(accessories):Fixing background and collapsible

AnneThKu committed:
fix(accessories):Fixing background and collapsible

AnneThKu committed:
fix(accessories): Fix for adding random images

AnneThKu committed:
fix(accessories): Fix for adding random images

AnneThKu committed:
fix(accessories): Changed class for brochure links

AnneThKu committed:
fix(accessories): Changed class for brochure links

AnneThKu committed:
fix(pulse):Fix for broken link for CNA news

AnneThKu commented:
> @AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?

When the archived version is added as link, the image opens as zoomed-in, and once the user clicks, it zooms out to a size that we can read. So, added as image in https://github.com/subnero1/subnero1.github.io/pull/120/commits/72d77b998fcbfe64b60297128564478ecd19e824. Please check and let me know if this looks ok.

What can be done for the remaining links?
- Can we use this link https://sites.google.com/edu.haifa.ac.il/anl/personnel for `Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff`?

>Will need @AnneThKu help with
Fix collapsing in the modem pages
Fixing bg color in accessories page to match with modem page
Having random number of images in accessories page without changing size

All the above mentioned issues are fixed. Please help to check.

AnneThKu committed:
fix(accessories):Removing collapse.html

AnneThKu committed:
fix(accessories):Removing collapse.html

AnneThKu committed:
fix(accessories):Fix for image width

AnneThKu committed:
fix(accessories):Fix for image width

manuignatius committed:
fix: Addressed comments from Chinmay and Mandar on the PR

manuignatius committed:
fix: Addressed comments from Chinmay and Mandar on the PR

manuignatius commented:
> @manuignatius
> Overall much better than what we have, so I recommend going ahead with this.
> 
> A critique, however, is that the content doesn't seem to flow smoothly, in general. While nothing incorrect with the text, the overall pages feel like disconnected text blurbs. At some point, we might need a more holistic look at the overall flow of the site.
> 
> For example, if you look at the multichannel modem page, it starts with "Subnero underwater modems provide options for customization and extension at many levels." But this is a standard configuration, so there's no customization involved! Then after key features, you have these 3 inset boxes which feel like they're meant to clickable to get more info, but they're not. And they just repeat what the last 3 bullets of features say.

Updated the multi-channel modem page to address your comments.

Yes, I agree, once we publish these (and may be after the next couple of minor updates), we need a holistic review of our entire website.

manuignatius commented:
> > @AnneThKu Can we link to the rendered PDFs/PNGs of the Internet archive versions for those links (if available)?
> 
> When the archived version is added as link, the image opens as zoomed-in, and once the user clicks, it zooms out to a size that we can read. So, added as image in [72d77b9](https://github.com/subnero1/subnero1.github.io/commit/72d77b998fcbfe64b60297128564478ecd19e824). Please check and let me know if this looks ok.
> 
> What can be done for the remaining links?
> 
> * Can we use this link https://sites.google.com/edu.haifa.ac.il/anl/personnel for `Node Modem taking the plunge (30-Mar-2020) - https://sites.google.com/edu.haifa.ac.il/anl/staff`?
> 
> > Will need @AnneThKu help with
> > Fix collapsing in the modem pages
> > Fixing bg color in accessories page to match with modem page
> > Having random number of images in accessories page without changing size
> 
> All the above mentioned issues are fixed. Please help to check.

@AnneThKu I tested all the image related comments from @notthetup and all seems to work in my setup. There is only 1 comment pending that I need your help with.

There are some `+` signs in the various modem pages (only in modem pages) which are meant to be made to collapsible. Can you apply the collapse feature to one of the pages so that I can adapt it to the rest of the pages? Pages are:
- wnc-m25mpe3.md
- wnc-m25mps3.md
- wnc-m25mrs3.md
- wnc-m25mse3.md
- wnc-m25mss3+xch.md
- wnc-m25mss3.md

manuignatius committed:
Updated modem brochure

manuignatius committed:
Updated modem brochure

AnneThKu committed:
feat(products): Adding the collapsible for tech specs

AnneThKu committed:
feat(products): Adding the collapsible for tech specs

AnneThKu committed:
fix(css): Fix for overflowing text

AnneThKu committed:
fix(css): Fix for overflowing text

AnneThKu committed:
fix(products):Making all tech specs and product information collapsed

AnneThKu committed:
fix(products):Making all tech specs and product information collapsed

AnneThKu committed:
fix(accessories): Fix for image urls

AnneThKu committed:
fix(accessories): Fix for image urls

AnneThKu committed:
fix(products):Fix for rendering useful links in wnc-m25mss3+xch.md

AnneThKu committed:
fix(products):Fix for rendering useful links in wnc-m25mss3+xch.md

AnneThKu committed:
feat(pulse):Adding post for the SwanViz UI

AnneThKu committed:
fix(pulse):Updated the post text, title and date

AnneThKu committed:
fix(support):Updated links for Relevant papers

Florie25 commented:
@manuignatius , here are my comments:
**Landing page:**
- pictures of Platinum edition modems should be centred.
- could we replaced the picture of the multichannel modem with a picture of a full modem. The current picture makes the cut really stand out.
- Can we change the banner to get rid of the node configuration?

**Primary Battery pack page:**
- Consider "on-site' instead of "in-field" replaceable
- typo under product information "rechargeable"

AnneThKu committed:
fix(pulse):fixing broken links in the posts

AnneThKu committed:
fix(pulse):External links to open in mew tab

AnneThKu committed:
fix(pulse):Fix for #151

AnneThKu committed:
Merge pull request #150 from subnero1/pulse-swanviz

Adding pulse post for the SwanViz UI and CSSSR

AnneThKu committed:
fix(pulse):Adding missed image

ImgBotApp committed:
[ImgBot] Optimize images

/images/pulse-CNA-SWAN-news.png -- 590.81kb -> 469.27kb (20.57%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

notthetup committed:
Merge pull request #152 from subnero1/imgbot

[ImgBot] Optimize images

manuignatius committed:
fix: Addressed florie's comments on PR

manuignatius committed:
fix: Addressed florie's comments on PR

manuignatius commented:
@Florie25 Addressed all your comments
- Centred the platinum modems (good catch)
- New image for multi-channel modem
- New banner image for modem page - removed node configuration
- Updated to "on-site"
- Fixed typo

manuignatius committed:
Updated products banner and WNC boxart images

manuignatius committed:
Updated products banner and WNC boxart images

manuignatius committed:
Adjusted hue of modem boxart image

manuignatius committed:
Adjusted hue of modem boxart image

dependabot[bot] committed:
Bump rexml from 3.2.4 to 3.2.5

Bumps [rexml](https://github.com/ruby/rexml) from 3.2.4 to 3.2.5.
- [Release notes](https://github.com/ruby/rexml/releases)
- [Changelog](https://github.com/ruby/rexml/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/rexml/compare/v3.2.4...v3.2.5)

Signed-off-by: dependabot[bot] <support@github.com>

notthetup committed:
Merge pull request #153 from subnero1/dependabot/bundler/rexml-3.2.5

Bump rexml from 3.2.4 to 3.2.5

manuignatius committed:
Updated transducer cable length from 1m to 5m for EC. Rest cable lengths unchanged

manuignatius committed:
Updated transducer cable length from 1m to 5m for EC. Rest cable lengths unchanged

manuignatius committed:
fix: primary battery pack to battery pack

manuignatius committed:
fix: primary battery pack to battery pack

manuignatius committed:
fix: tranducer cable length fix

manuignatius committed:
fix: tranducer cable length fix

AnneThKu committed:
fix(pulse):Fixing merge conflicts

manuignatius committed:
Create 2021-05-12-Subnero-Multi-channel-Modems.md

AnneThKu committed:
fix(pulse):Adding header to the content and target attribute to external links

manuignatius committed:
Merge pull request #156 from subnero1/multi-channel-pulse-article

Multi-channel-Modems pulse article

manuignatius committed:
Update 2021-05-12-Subnero-Multi-channel-Modems.md

manuignatius committed:
fix: Fixed images to remove srewheads

manuignatius committed:
fix: raw recording - removed reference to passband and baseband

AnneThKu committed:
feat(products):Added announcement texts in landing page and new tags to products

AnneThKu committed:
fix(products):Fixing text width in landing page

notthetup reviewed (approved):
Looks good. Only one comment, the

> CONTACT US FOR A QUOTE

on each of the product pages seems very awkwardly placed between the product description and specs. Maybe another way to position that would be nice?

manuignatius commented:
Good catch @notthetup , that was supposed to be a button, the text was only a placeholder. Let me see how I can update it.

manuignatius committed:
fix: Updated to use a button for 'Get a quote' text

manuignatius committed:
Updated pulse post to include the video

manuignatius committed:
Added brochure to multi-channel page

AnneThKu committed:
fix(products):Fix for collapsible part of useful links

AnneThKu committed:
fix(products): Modified the excerpts to mention underwater acoustic modems

AnneThKu committed:
fix(products):Modified useful links related to site.baseurl

manuignatius committed:
Updated landing page image lp1.jpg

AnneThKu committed:
fix(products):Removed the CI issue link

manuignatius committed:
feat: Added product launch video to the product page
